operations/template/app.tf

```diff
@@ -137,8 +137,7 @@ resource "azurerm_linux_web_app" "api" {
     }
   }
 
-  #   When adding new settings that are needed for the live app but shouldn't be used in the pre-live
-  #   slot, add them to `sticky_settings` as well as `app_settings` for the main app resource
+  # New settings may also need to be added to the pre-live slot's app_settings
```

Comment:
Can we expand this comment to include when to add to both as opposed to just one.

Reply:
We updated the comment to just say to always add them to both - for TI I think this is cleaner